pymatgen/io/lobster/lobsterenv.py

```diff
@@ -1134,10 +1143,10 @@ def _get_limit_from_extremum(
 
         Args:
             icohpcollection: icohpcollection object
-            percentage: will determine which ICOHPs will be considered (only 0.15 from the maximum value)
+            percentage: will determine which ICOHPs/ICOOP/ICOBI will be considered (only 0.15 from the maximum value)
             adapt_extremum_to_add_cond: should the extrumum be adapted to the additional condition
             additional_condition: additional condition to determine which bonds are relevant
-        Returns: [-100000, min(max_icohp*0.15,-0.1)]
+        Returns: [-inf, min(max_icohp*0.15,noise_cutoff)] / [max(max_icohp*0.15, noise_cutoff),inf]
```

Comment:
I think you mixed up plus and minus here.

Reply:
Yeah , will fix this

Reply:
Fixed the error in doc-string

Reply:
Maybe put (ICOHP) and (ICOOP, ICOBI) in brackets? Not sure people will directly get the "/". I would also replace "/" with or.

Reply:
Maybe, we should also say `max_abs_icohp`. Otherwise, very good :).

Reply:
changed doc-string to `strongest_icohp` as per your suggestion 😃